includes/Modules/AdSense.php

```diff
@@ -259,10 +269,14 @@ protected function output_adsense_script( $client_id ) {
 		// If we haven't completed the account connection yet, we still insert the AdSense tag
 		// because it is required for account verification.
 		?>
-<script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script> <?php // phpcs:ignore WordPress.WP.EnqueuedResources.NonEnqueuedScript ?>
+<script
+	async
+	src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js" <?php // phpcs:ignore WordPress.WP.EnqueuedResources.NonEnqueuedScript ?>
+		<?php echo $this->get_tag_block_on_consent_attribute(); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
```

Comment:
Not sure why but PHPCS insists on this having 2 indents rather than 1.

Reply:
I think we can keep this in a single line. This will also ensure we don't add extra whitespace. The opening `<?php` tag for conditionally adding the attribute can be right after the previous closing `"`, as it should add an initial space as needed.